wpe/src/main/cpp/Browser/ExportedBuffer.h

```diff
@@ -57,7 +44,22 @@ struct ExportedBuffer final : public std::enable_shared_from_this<ExportedBuffer
 
     ~ExportedBuffer()
     {
-        if (buffer)
-            AHardwareBuffer_release(buffer);
+        if (m_buffer != nullptr)
+            AHardwareBuffer_release(m_buffer);
     }
+
+    AHardwareBuffer* getBuffer() const noexcept { return m_buffer; }
+    uint32_t getPoolId() const noexcept { return m_poolId; }
```

Comment:
not according to style guide

https://webkit.org/code-style-guidelines/#names-setter-getter

Reply:
done